packages/mr/README

```diff
@@ -0,0 +1,17 @@
+# [Office UI Fabric React Mr](http://dev.microsoft.com/fabric)
+
+##### Mr components for Office UI Fabric React
```

Comment:
As @JasonGore points out in his comment, this should be clarified to something like "Mixed-reality components for..."